AutomatedTesting/Gem/PythonTests/prefab/PrefabLevel_BasicWorkflow.py

```diff
@@ -0,0 +1,66 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+# fmt:off
+class Tests():
+    create_new_entity =  ("Entity: 'CreateNewEntity' passed",                          "Entity: 'CreateNewEntity' failed")
+    create_prefab =      ("Prefab: 'CreatePrefab' passed",                             "Prefab: 'CreatePrefab' failed")
+    instantiate_prefab = ("Prefab: 'InstantiatePrefab' passed",                        "Prefab: 'InstantiatePrefab' failed")
+    new_prefab_pos =     ("Prefab: new prefab's position is at the expected position", "Prefab: new prefab's position is *not* at the expected position")
```

Comment:
repeated nit : try to use 'position' everywhere instead of 'pos'

Reply:
Fixed.